.changeset/six-eels-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Fixes a variable name typo
```

Comment:
We don't need a changeset because it's not a feature, bug fix, or significant refactor.